tests/providers/google/cloud/hooks/test_kubernetes_engine.py

```diff
@@ -20,12 +20,13 @@
 import sys
 from asyncio import Future
 
+import kubernetes.client
 import pytest
 from google.cloud.container_v1 import ClusterManagerAsyncClient
 from google.cloud.container_v1.types import Cluster
 
 from airflow.exceptions import AirflowException
-from airflow.providers.google.cloud.hooks.kubernetes_engine import AsyncGKEHook, GKEHook
+from airflow.providers.google.cloud.hooks.kubernetes_engine import AsyncGKEHook, AsyncGKEPodHook, GKEHook
```

Comment:
```suggestion
from airflow.providers.google.cloud.hooks.kubernetes_engine import GKEAsyncHook, GKEPodAsyncHook, GKEHook
```
naming should be inline with other async hooks across the repo

Reply:
Done